includes/abilities-api/class-wp-abilities-registry.php

```diff
@@ -56,8 +56,8 @@ final class WP_Abilities_Registry {
 	 *   description?: string,
 	 *   input_schema?: array<string,mixed>,
 	 *   output_schema?: array<string,mixed>,
-	 *   execute_callback?: callable( array<string,mixed> $input): (mixed|\WP_Error),
-	 *   permission_callback?: ?callable( array<string,mixed> $input ): (bool|\WP_Error),
+	 *   execute_callback?: callable( mixed $input= ): (mixed|\WP_Error),
+	 *   permission_callback?: ?callable( mixed $input= ): (bool|\WP_Error),
```

Comment:
And elsewhere I think

```suggestion
	 *   execute_callback?: callable( mixed $input ): (mixed|\WP_Error),
	 *   permission_callback?: ?callable( mixed $input ): (bool|\WP_Error),
```

Reply:
It's intentional to account for the case where we don't pass the first argument if there is no input schema and the value is `null`.

Reply:
Never seen that syntax before. Neat!

Reply:
<img width="1964" height="1522" alt="image" src="https://github.com/user-attachments/assets/23f34893-10a3-4860-85e4-3aff66984673" />